### PR TITLE
fix issue-197

### PR DESCRIPTION
**Pull request by lixiaoqity**
### Fixes #<197>

<Do not delete this template>
Description: <provide a description> The title should be short and relevant.

#197 Adding a title "Communities of House of Geeks" for the Communities Page.

#### Checklist

- [Y ] I've followed the Contributing guidelines provided in the repository.

- [Y ] I've made the changes which were demanded in the linked issue.

- [Y ] I've tested my code on a Chromium based browser.

- [Y ] I've tested my code on Mozilla Firefox.

- [Y ] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>
image

### CODE REVIEW

File: client/src/components/Community/Community.css [hunk 1]
```diff
@@ -20,12 +20,10 @@
 }
 
 .community-component .title {
-  opacity: 0;
-  width: 0;
-  height: 0;
-  pointer-events: none;
-  margin: 0;
-  padding: 0;
```
Comment (EmperorYP7) [inline client/src/components/Community/Community.css:28]: Retain these properties.

File: client/src/components/Community/Community.css [hunk 2]
```diff
@@ -20,12 +20,10 @@
...
+  color: #ee2a7b;
+  font-size: 8vh;
+  text-align: center;
+  margin-top: 3em;
```
Comment (EmperorYP7) [inline client/src/components/Community/Community.css]: I think we dont need any margin there

File: client/src/components/Community/Community.js [hunk 3]
```diff
@@ -9,9 +9,9 @@ function Community() {
   return (
     <div className="community-component">
       <div className="side-banner">Wings of House of Geeks</div>
+      <div className="title">Communities of House of Geeks</div>
       <div className="container-fluid main">
-        <div className="row justify-content-around">
-          <div className="title">Communities of House of Geeks</div>
```
Comment (EmperorYP7) [inline client/src/components/Community/Community.js:14]: Retain this. Don't add a new tag for this. Try to manipulate the present tags and classes to solve this issue

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
You do have to work around this a bit. Don't add or remove any tags though. Thanks for your contribution! :smile:

File: client/src/components/Community/Community.css (hunk 1 above)
Comment (lixiaoqity) [reply client/src/components/Community/Community.css:28]: I have made some changes on the branch. I keep these properties, but also add some new properties.

File: client/src/components/Community/Community.css (hunk 2 above)
Comment (lixiaoqity) [reply client/src/components/Community/Community.css]: I cancelled this margin-top property. But I assign a value in the old margin property to separate the <div> with others below.

**Comment by tusharjain0022** [comment]
Heading shouldn't be visible on the home page. Heading should render when we are on the `/community` section when we visit the community section from the navbar. And it would look nice after reducing the margin-top .

Screenshot (56)

**Comment by tusharjain0022** [comment]
The heading shouldn't be visible on the home page, it should be visible when the community component is rendered on the `/community` page when we visit the community section via navbar.


To achieve this try adding these👇 lines before the render function
`const currentloc = String(window.location); //Gets current URL`
 `let currenttab = currentloc.substring(currentloc.lastIndexOf("/") + 1); //Gets current element from navbar`


And use conditional rendering using something like this👇
` {currenttab == "community" && (
            <div className="title">Communities of House of Geeks</div>
          )}`

Screenshot (56)

File: client/src/components/Community/Community.css [hunk 4]
```diff
@@ -20,12 +20,15 @@
...
 
 .community-component .title {
-  opacity: 0;
-  width: 0;
+  opacity: 1;
+  width: 20em;
   height: 0;
   pointer-events: none;
-  margin: 0;
+  margin: 3em;
```
Comment (tusharjain0022) [inline client/src/components/Community/Community.css]: try changing the margin property to `margin-bottom` ,as currently a lot of margin is there on the top of the heading

Comment (lixiaoqity) [reply client/src/components/Community/Community.css]: All right, I have fixed it.

File: client/src/components/Community/Community.js [hunk 5]
```diff
@@ -6,12 +6,14 @@ import list from "./CommunityList";
...
+  let currenttab = currentloc.substring(currentloc.lastIndexOf("/") + 1); 
   return (
     <div className="community-component">
-      <div className="side-banner">Wings of House of Geeks</div>
+      <div className="side-banner">Wings of House of Geeks</div>      
       <div className="container-fluid main">
-        <div className="row justify-content-around">
-          <div className="title">Communities of House of Geeks</div>
+        <div className="row justify-content-around"> 
+          {currenttab == "community" && ( <div className="title">Communities of House of Geeks</div> )}
```
Comment (tusharjain0022) [inline client/src/components/Community/Community.js]: replace `==` with `===`

**Review by tusharjain0022 — CHANGES_REQUESTED** [review]

File: client/src/components/Community/Community.js (hunk 5 above)
Comment (lixiaoqity) [reply client/src/components/Community/Community.js]: Thanks. Have fixed it.

**Review by hariom1625 — COMMENTED** [review]
There is some unusual margin present above the community title, have a look at it too @lixiaoqity.

File: client/src/components/Community/Community.js (hunk 3 above)
Comment (lixiaoqity) [reply client/src/components/Community/Community.js:14]: I have erased the tap and space.Just add a judge condition. Thanks.

**Comment by ankiiitraj** [comment]
@hariom1625 is this ready to be merged?

**Comment by hariom1625** [comment]
The changes which I requested isn't made yet.
@ankiiitraj @lixiaoqity  I was talking about the margin present here. 
Screenshot (212)

**Comment by tusharjain0022** [comment]
@lixiaoqity any updates on this?

**Comment by lixiaoqity** [comment]
I adjust the community title close to the top at the community page.

**Review by EmperorYP7 — APPROVED** [review]
LGTM

**Review by tusharjain0022 — APPROVED** [review]
LGTM! 💯

**Comment by ankiiitraj** [comment]
@all-contributors please add @lixiaoqity  for code

**Pull request merged**